inception-workload/src/main/java/de/tudarmstadt/ukp/inception/workload/extension/WorkloadExtension.java

```diff
@@ -15,16 +15,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package de.tudarmstadt.ukp.inception.workload.dynamic.manager;
+package de.tudarmstadt.ukp.inception.workload.extension;
 
-/**
- * @deprecated This needs to be replaced by a traits object for the dynamic workload manager as soon
- *             as possible.
- */
-@Deprecated
-public interface DefaultAnnotationsProperties
-{
-    int getNumberOfDefaultAnnotations();
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.clarin.webanno.support.extensionpoint.Extension;
 
-    void setNumberOfDefaultAnnotations(int aAnnotations);
+public interface WorkloadExtension extends Extension<Project>
+{
+    @Override
+    default boolean accepts(Project project)
+    {
+        return true;
```

Comment:
Why do you use a default?

Reply:
Otherwise I have to implement accepts() in all the other ExtensionPoints, and they all would have a "return = true" in them, so I thought it would be smoother to implement it only once in the interface